src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -131,6 +132,55 @@ public int getDocid() {
     }
   }
 
+  public static class IndexTerm {
```

Comment:
I think just `Term` is fine?

Reply:
`Term` clashes with `org.apache.lucene.index.Term`, but we could use it with the package name if we really want `Term`?

Reply:
sg, keep as `IndexTerm`